swap_meet/item.py

```diff
@@ -1,2 +1,52 @@
+import uuid
+
+# Wave 2- SJ
+# 1. each Item will have an attribute, id -- which is a unique integer by default
+# 2. import uuid
+#    using UUID4 since UUID1 creates a UUID with the computer's network address 
+#    UUID.int- returns the UUID as a 128-bit integer.
+#    self.id = uuid.uuid4().int
+
+# following a similar syntax for condition as id
+
+
+
 class Item:
-    pass
\ No newline at end of file
+    
+    def __init__(self, id=None, condition = 0):
+
+        self.id =  id if id is not None else uuid.uuid4().int
```

Comment:
Something to consider, what happens here if someone tries to pass in an `id` that is not `None`, but is an `id` that is already assigned to something in inventory? What type of logic could we implement here in order to prevent that from happening?